### DISCUSSION TITLE
Use custom grammar rules on preview

### DISCUSSION THREAD

**Discussion by ryu-man** [BearToCode/carta#102] [General]
Hi there, I made some custom grammar rules, and they are working great in the editor but when I switch to the preview tab, those rules seem to have no effect, Is there any idea how to make them work for both the editor and preview modes?

**Comment by BearToCode** [comment]
Hi, can you explain a bit more what are you trying to achieve? Did you add custom grammar rules for Markdown highlighting or something else? In the first case, I don't see what you expect to see in the preview tab.
